src/components/create-card.jsx

```diff
@@ -1,7 +1,42 @@
 import React from "react";
 
 function CreateCard(props) {
-  return <h1 className="text-center">Create New Card</h1>
+  function handleSubmit(event) {
+    event.preventDefault();
+    const formQuestion = event.target.question.value;
+    const formAnswer = event.target.answer.value;
+    const card = {
+      question: formQuestion,
+      answer: formAnswer
+    }
+    props.addCard(card);
+    handleReset(event);
+  }
+
+  function handleReset(event) {
+    event.target.reset();
+    props.setView("view-cards");
+  }
```

Comment:
We usually don't want to define a function within another function, also the inputs values should be controlled by state. So the best option is to use a class component here. This will allow you to have some helper methods and also have state for the inputs.

Reply:
ok, I was wondering if converting to class was the right option, but I got distracted by my burgeoning understanding of how to handle form submits XD